common/source/docs/common-temperature-sensor.rst

```diff
@@ -5,54 +5,60 @@
 Temperature Sensors
 ===================
 
-ArduPilot supports up to nine individual temperature sensors with I2C interfaces:
+.. note:: Released firmware for most autopilots does not include temperature sensor support by default. You may need to use the `Custom Firmware Builder <https://custom.ardupilot.org>`__ and enable ``Sensors/Temperature Sensors`` as well as one or more of ``Sensors/Temp Sensor - <TYPE>`` to use this feature on your autopilot. If using an analog sensor, enable any one (or more) of the I2C devices in the build menu to ensure the backend is properly included.
```

Comment:
this is incorrect....all bds with 2MB or more have several temp sensors included...and in general I dont like warning about feature inclusion/exclusion except in the firmware limitations page due to maintenance problems that are caused since they are in continuous flux as the code evolves...so I think this should be removed

Reply:
We may have a bigger issue, then (and this was the primary motivation for the PR):

`AP_TEMPERATURE_SENSOR_ENABLED = 2` is what those boards get. It enables dummy drivers, shows enabled in `features.txt`, but does nothing. Tested with 4.5.5 and CubeOrangePlus today. Even to get analog sensor support, I had to either explicitly `define AP_TEMPERATURE_SENSOR_ENABLED 1` in the hwdef or use the Custom Firmware Builder  and enable an I2C sensor in addition to the basic backend in order to get any TEMPx parameter to display.

There are only 2 periph boards that actually include temperature sensor support in the released firmware.

Reply:
okay then the code needs fixing,not the wiki....needs a code bug report....someone assumed having a bool enable set to 2 sets the normal temp sensors backend includes to test true

Reply:
I agree. Some Discord discussion with @peterbarker and @IamPete1 drove me to add the note. Will raise the issue. Note removed.